rumqttd/src/lib.rs

```diff
@@ -102,10 +102,8 @@ pub struct BridgeConfig {
 #[derive(Debug, Serialize, Deserialize, Clone)]
 pub struct ConnectionSettings {
     pub connection_timeout_ms: u16,
-    pub throttle_delay_ms: u64,
     pub max_payload_size: usize,
-    pub max_inflight_count: u16,
-    pub max_inflight_size: usize,
+    pub max_inflight_count: usize,
```

Comment:
I think we should update this variable name to a more descriptive `max_connection_buffer_len` that we use internally in `Network`, so it is not confused with max allowed inflight publish messages which is currently a constant set to 100 (i.e. MAX_INFLIGHT in `src/router/iobufs.rs`).